media/css/protocol/components/callout.scss

```diff
@@ -0,0 +1,82 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/callout';
+
+// Temporary styles for call-out while the hero component has been depreciated. Will be updated with a new component
+.mzp-c-callout.mzp-t-hero {
+    .mzp-l-content {
+        box-sizing: border-box;
+        max-width: $content-sm;
+        padding: $layout-md $layout-xs;
+    }
+
+    .mzp-c-callout-title {
+        @include text-title-lg;
+    }
+
+    @media #{$mq-md} {
+        .mzp-l-content {
+            max-width: $content-md;
+            padding: $layout-md $layout-xl;
+        }
+    }
+
+    @media #{$mq-lg} {
+        .mzp-l-content {
+            padding: $layout-xl 80px;
+        }
+    }
+}
+
+.mzp-c-callout[class*='mzp-t-product-'].mzp-t-hero .mzp-c-callout-title {
+    background-size: 80px 80px;
+}
+
+.mzp-c-callout {
+    .mzp-l-content {
+        .mzp-t-callout-brand {
+            margin: $spacing-md auto;
+
+            .mzp-c-callout-title {
+                padding-top: $spacing-lg;
+            }
+
+        }
+    }
+}
+
+.mzp-l-compact {
+    .mzp-l-content {
+        .mzp-c-callout-content:has(.mzp-t-callout-compact-brand) {
```

Comment:
With the new update to Protocol's Callout component, logos/wordmarks aren't going to be natively integrated. Since not all callouts/compact callouts had logos and icons, I had to make sure some classes had certain rules _only if_ they contained branding classes as children. It was a new and interesting use case I had to code for, so I opted to use the newly supported [`:has()` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:has). However, since it's only _newly_ supported, it would be a safer idea to come up with a way to write the conditional rules differently. I'll dive deeper into this later into the PR

Reply:
Update: I found a much easier way around this. I just added conditional that specified if the `brand` parameter is true, then add the class name of `mzp-c-callout-content-has-brand` to the parent container. This made styling much easier and will work for all browsers 👍🏼 

[Here's the commit ](https://github.com/mozilla/bedrock/pull/14084/commits/e702da321fcb4051b54066af16621398bd063c43)to be more specific